ldes-server-admin/src/test/java/be/vlaanderen/informatievlaanderen/ldes/server/admin/domain/eventstream/http/valueobjects/EventStreamResponseTest.java

```diff
@@ -24,36 +24,32 @@ class EventStreamResponseTest {
 	private static final String COLLECTION = "collection";
 	private static final String TIMESTAMP_PATH = "generatedAt";
 	private static final String VERSION_OF_PATH = "isVersionOf";
-	private static final String MEMBER_TYPE_PATH = "memberType";
 	private static final boolean HAS_DEFAULT_VIEW = false;
```

Comment:
I see that `HAS_DEFAULT_VIEW` is not used anymore. This is probably from another PR, but could you remove it please?